.github/workflows/install.yml

```diff
@@ -1,5 +1,9 @@
 name: Install to Ubuntu
 on:
+  # TODO: Added to verify that the CI works. We will delete it after verification.
+  pull_request:
+    paths:
+      - ".github/workflows/install.yml"
```

Comment:
Could you remove this before we merge this?
We could confirm that this works.

Reply:
Sure. I will remove it!

Reply:
fix: [7d03022](https://github.com/mroonga/mroonga/pull/691/commits/7d0302280d611e77c232d46dc08056bcb03b5181) I've just removed it.